next-version/action.yml

```diff
@@ -33,3 +37,5 @@ runs:
     - ${{ inputs.tag-prefix }}
     - --output-prefix
     - ${{ inputs.output-prefix }}
+    - --fail
+    - ${{ inputs.fail }}
```

Comment:
I might be remembering incorrecty, but I think this syntax will not work for `BoolFlags` (urfave/cli expects you to use `--fail=0`). Can we add an integration test covering this flag to confirm it?

Reply:
sure

Reply:
Fixed in last commit and test added.